frontend/src/components/ExpensePage.js

```diff
@@ -1,21 +1,50 @@
-import React from "react";
+import React, { Component } from "react";
 import Header from "./Header";
 import ExpensePresentation from "./ExpensePresentation";
 import ExpenseList from "./ExpenseList";
 import Footer from "./Footer";
+
 import "../styles/ExpensePage.css";
 
-function ExpensePage() {
-  return (
-    <div className="ExpensePage">
-      <Header />
-      <main>
-        <ExpensePresentation />
-        <ExpenseList />
-      </main>
-      <Footer />
-    </div>
-  );
+export class ExpensePage extends Component {
+  state = {
+    total: 0,
+  };
+
+  updateTotal = (amount) => {
+    this.setState({ total: this.state.total + parseFloat(amount) });
+  };
+
+  // React Life Cycle
+  componentDidMount() {
+    this.total = JSON.parse(localStorage.getItem("total"));
```

Comment:
"total" is now being saved both directly in the component object root (ExpensePage.total) _and_ state (ExpensePage.state.total). We should use only state. The local storage call can be e.g. put directly in the following setState call.

Reply:
Need more clarity

Reply:
Made changes as requested.